format/Message.fbs

```diff
@@ -109,12 +134,16 @@ table Field {
   dictionary: long;
   // children apply only to Nested data types like Struct, List and Union
   children: [Field];
+  /// layout of buffers produced for this type (as derived from the Type)
+  /// does not include children
```

Comment:
This is `Field` which is part of the `Schema` (`FieldNode` and not `Field` will contain all the `Buffers` for a `RecordBatch`).
Each `Field` has a vector layout and children. Each child define its own layout.
For example:
 - a `Tuple` only has a validity vector (children define their own).
 - a `List` has a validity vector and an offset vector (and a single child)
 - an int has a validity vector and a data vector (and no children)